spec/commands/remove_spec.rb

```diff
@@ -0,0 +1,98 @@
+# frozen_string_literal: true
+
+RSpec.describe "bundle remove" do
+  context "remove a single gem from gemfile" do
+    it "when gem is present in gemfile" do
+      gemfile <<-G
+        source "file://#{gem_repo1}"
+
+        gem "rack"
+      G
+
+      bundle "remove rack"
+
+      expect(gemfile).to_not match(/gem "rack"/)
```

Comment:
can you please change these tests to check for equality? That'll make reviewing much easier

Reply:
I was thinking about writing a test something like this
```ruby
expected_gemfile = "source \"file://#{gem_repo1}\"\n"
expect(gemfile).to eq(expected_gemfile)
```

for some test, directly matching the gemfile? Is it fine to match this way?